TypeScript_ESLint.snb.md

```diff
@@ -111,21 +195,25 @@ https://sourcegraph.com/github.com/typescript-eslint/typescript-eslint@3eab88902
+The `converter` method it invokes is co-recursive with `convertNode`, which is just a giant switch statement on the AST node type that constructs the appropriate ESLint node corresponding to the current TypeScript node.
+The two corresponding nodes are then added to the TypeScript-to-ESLint node mapping:
 
 https://sourcegraph.com/github.com/typescript-eslint/typescript-eslint@3eab889022c9d1617f275017d6951f663ea57f24/-/blob/packages/typescript-estree/src/convert.ts?L127-132
 
 https://sourcegraph.com/github.com/typescript-eslint/typescript-eslint@3eab889022c9d1617f275017d6951f663ea57f24/-/blob/packages/typescript-estree/src/convert.ts?L770-836&subtree=true
 
-In this fashion, the TypeScript AST is converted by recursively traversing the AST and building the new ESLint one along the way. Both ASTs and the mapping between them are stored so they can be referenced by rules.
+In this fashion, the TypeScript AST is converted by recursively traversing the AST and building the new ESLint one along the way.
+Both ASTs and the mapping between them are stored so they can be referenced by rules.
 
 ## Next steps
 
-TypeScript ESLint is an amazing tool that brings the power of static analysis to multitudes of TypeScript projects. It is also an impressive case study of code reuse. Rather than build an entire linting project from scratch, it finds an elegant way to build on top of JavaScript's ESLint while integrating the TypeScript compiler to enable rules to take advantage of type information.
+TypeScript ESLint is an amazing tool that brings the power of static analysis to multitudes of TypeScript projects.
+It is also an impressive case study of code reuse.
+Rather than build an entire linting project from scratch, it finds an elegant way to build on top of JavaScript's ESLint while integrating the TypeScript compiler to enable rules to take advantage of type information.
 
 There are many ways to get involved:
 
-* [See what sorts of rules and autofixes it can apply in the playground](https://typescript-eslint.io/play/)
-* [Use it for your TypeScript codebase](https://typescript-eslint.io/docs/linting/)
-* [Contribute a new linting rule](https://typescript-eslint.io/docs/development/custom-rules)
-* [Support this open-source project financially](https://opencollective.com/typescript-eslint/contribute)
+- [See what sorts of rules and autofixes it can apply in the playground](https://typescript-eslint.io/play/)
+- [Use it for your TypeScript codebase](https://typescript-eslint.io/docs/linting/)
+- [Contribute a new linting rule](https://typescript-eslint.io/docs/development/custom-rules)
+- [Support this open-source project financially](https://opencollective.com/typescript-eslint/contribute)
```

Comment:
Pardon the unnecessary format changing; my Prettier plugin is very aggressive. 😅